src/ResourceManagement/AppService/HostNameBindingImpl.cs

```diff
@@ -140,7 +140,7 @@ private string NormalizeHostNameBindingName(string hostname, string domainName)
         ///GENMHASH:027705B70337BE533ED77421800E496A:3479885A232C974264B5B36BDBB0BC94
         public HostNameBindingImpl<FluentT, FluentImplT, DefAfterRegionT, DefAfterGroupT, UpdateT> WithDnsRecordType(CustomHostNameDnsRecordType hostNameDnsRecordType)
         {
-            var regex = new Regex("([.\\w-]+)\\.([\\w-]+\\.\\w+)");
+            var regex = new Regex("([.\\w-]+|[*])\\.([\\w-]+\\.\\w+)");
```

Comment:
It is better not to add `[` and `]`.

Reply:
It seems to work fine in my environment. I guess you got an error cause dns zone haven't been setup accordingly
![Screenshot 2020-11-18 141755](https://user-images.githubusercontent.com/31245557/99524042-ebaa8e80-29a8-11eb-83e0-b75c5840444a.png)

Reply:
@SvyatovetsD
Could you see if you can apply @ChenTanyi 's advice above?